parsec/core/fs/userfs/userfs.py

```diff
@@ -366,7 +366,13 @@ async def workspace_create(self, name: AnyEntryName) -> EntryID:
             user_manifest = user_manifest.evolve_workspaces_and_mark_updated(workspace_entry)
             await self._create_workspace(workspace_entry.id, workspace_manifest)
             await self.set_user_manifest(user_manifest)
-            self.event_bus.send(CoreEvent.FS_ENTRY_UPDATED, id=self.user_manifest_id)
+            self.event_bus.send(
+                CoreEvent.FS_ENTRY_UPDATED,
+                id=self.user_manifest_id,
+                reason=FSEntryUpdatedReason.WORKSPACE_CREATE,
+                entry_id=workspace_entry.id,
```

Comment:
should call this `workspace_id`/`workspace_name` (given `entry` is a more generic term that we'd better use in the case we deal with file/folder entry)